core/config/toml/types.go

```diff
@@ -1301,3 +1304,83 @@ func (t *ThresholdKeyShareSecrets) validateMerge(f *ThresholdKeyShareSecrets) (e
 
 	return err
 }
+
+type Tracing struct {
+	Enabled         *bool
+	CollectorTarget *string
+	NodeID          *string
+	SamplingRatio   *float64
+	Attributes      map[string]string
```

Comment:
to get the toml test to pass you need an omit empty here on Attributes.

The problem is that CoreDefaults is initializing an empty map and with the tag it will be serialized in the txtar test

Attributes      map[string]string `toml:",omitempty"`

Aside: you can hack `TestScripts` in `chainlink/main_test.go` to only run `testdata/scripts/node/validate` and run something like`go test -timeout 300s -run ^TestScripts$ github.com/smartcontractkit/chainlink/v2 -v -race` to run the txtar test.

[i pulled your branch, reproduced the test failure as described, and fixed it by adding the tag]

Reply:
Ah, thank you!